### PR TITLE
Remove the #call directive

### PR DESCRIPTION
**Pull request by asottile**
- It is buggy: #188
- It requires code patterns that are unsafe when copy pasted (example taken from some yelpy code)
```python
# EXPLANATORY NOTE: `content` is a string of template output from between
# #call...#end, but without the Markup blessing.  We always trust template
# text, so each of these functions re-blesses.  Don't do that elsewhere.
def layout_column(content, column, id=None, classname=''):
    content = Markup(content)
    # ...
```
- The `#with` directive provides the same functionality

### CODE REVIEW

File: Cheetah/legacy_parser.py [hunk 1]
```diff
@@ -137,24 +137,22 @@ def _unescapeDirectives(s):
...
-    'call': None,
-    'while': None,
-    'for': None,
-    'if': None,
-    'try': None,
-    'filter': None,
-    'with': None,
-}
+CLOSABLE_DIRECTIVES = frozenset({
+    'block', 'compiler-settings', 'def', 'for', 'if', 'try', 'while', 'with',
```
Comment (bukzor) [inline Cheetah/legacy_parser.py]: 'block' is the only directive that's closable but not indenting?

**Comment by bukzor** [comment]
:shipit:

File: Cheetah/legacy_parser.py (hunk 1 above)
Comment (asottile) [inline Cheetah/legacy_parser.py]: and `#def`, idk why `#compiler-settings` is indenting it really shouldn't be :)

That list might actually be somewhat bogus, I'm going to try taking some things out and seeing what fails

Comment (asottile) [inline Cheetah/legacy_parser.py]: causes tests to fail for some reason or another.  I think the names of the constants are actually bogus :(  for another day!

**Pull request merged**